lib/demangler/llvm.ts

```diff
@@ -32,9 +32,8 @@ import {BaseDemangler} from './base.js';
 import {PrefixTree} from './prefix-tree.js';
 
 export class LLVMIRDemangler extends BaseDemangler {
-    // Identifiers can be quoted: https://llvm.org/docs/LangRef.html#identifiers
-    llvmSymbolRE = /@(?<symbol>[\w$.]+)/gi;
-    llvmQuotedSymbolRE = /@"(?<symbol>[^"]+)"/gi;
+
+    llvmSymbolRE = /@"?([^\s"(]+)/g;
```

Comment:
I had this fix stashed during a week of vacation, and I see now a different fix was merged. 
Maybe it's better to use a single regex, but *definitely* the quotes must be a part of the capture.